src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  spinner: boolean;
+  onDeleteTodo: (todoId: number) => void;
+  isDeleting: boolean;
+  deletingCompleted: boolean;
```

Comment:
You don't need this prop. You can use `isDeleting` only.

Reply:
Not fixed